src/cbc_sdk/platform/audit.py

```diff
@@ -39,3 +67,275 @@ def get_auditlogs(cb):
         """
         res = cb.get_object("/integrationServices/v3/auditlogs")
```

Comment:
Is there a plan to replace this with `_queue`?

Reply:
Yes, that would be story CBAPI-5259.